program-runtime/src/lib.rs

```diff
@@ -9,6 +9,7 @@ pub mod log_collector;
 pub mod native_loader;
 pub mod neon_evm_program;
 pub mod pre_account;
+pub mod runtime_config;
```

Comment:
This is pretty bike-sheddy, but I rather think this module should be in `solana-runtime`. Change my mind?

Reply:
Nope!  Right as I was finishing, the thought crossed my mind.